recipes/rb-pastel/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "pastel" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: f577708463313119d2f7303eefb24217879d188e94f2eea8a19373fdcee48354
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install --norc -l -V --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+    - rb-equatable >=0.6.0,<0.7.0
```

the gem requirement was `~> 0.6`, which maps to `>=0.6,<1`
```suggestion
    - rb-equatable >=0.6,<1
```